03_letters/brvishnu92/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := `a:2
+b:1
+`
+	actual := buf.String()
+	assert.Equalf(t, expected, actual, "got %v want %v", actual, expected)
+}
+
+func TestLetters(t *testing.T) {
+	testCases := map[string]struct {
+		input    string
+		expected map[rune]int
+	}{
+		"test1":   {input: "blahbla", expected: map[rune]int{97: 2, 98: 2, 104: 1, 108: 2}},
+		"test2":   {input: "smarty", expected: map[rune]int{97: 1, 109: 1, 114: 1, 115: 1, 116: 1, 121: 1}},
+		"empty":   {input: "", expected: map[rune]int{}},
+		"sameval": {input: "aaaa", expected: map[rune]int{97: 4}},
+		"oneval":  {input: "a", expected: map[rune]int{97: 1}},
+	}
```

Comment:
Please add a test with non-ascii unicode runes in the string - this makes sure your unit under test is treating the elements of a string as runes and not bytes (which can be a common mistake for people coming from other languages).

Reply:
Done. Added special char and emojis